src/App.css

```diff
@@ -1,38 +1,288 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+body {
+  background-color: rgb(235, 233, 230);
+}
+
 .App {
-  text-align: center;
+  width: 100%;
+}
+
+.book-section {
+  align-items: center;
+  width: 100%;
+}
+
+.book-section-one {
+  width: 90%;
+  background-color: #fff;
+  height: 11rem;
+  margin-left: 3%;
+  padding: 20px;
```

Comment:
- [ ] Great effort working on the project styling, But currently there somethings you need to do so as to have a perfect replica of the origin design. Below is a video recording of some places where your attention is needed in the project. I would kindly suggest you work on them and be sure sure you have a perfect before submitting for review.

Video Claim 👇 

[screen-capture.webm](https://github.com/mailsg/Bookstore/assets/78634841/59b32868-e6e8-4f7d-9ef2-a9495148007e)

Reply:
@brainconnect93 the video is not working. Kindly check and let me know where do I need to make changes. 
![image](https://github.com/mailsg/Bookstore/assets/105475440/5e4d2b99-8b7a-4e54-ba23-9cc61d36f33b)